public/API/API_GetAchievementDistribution.php

```diff
@@ -16,7 +16,7 @@
 
 $gameID = (int) request()->query('i');
 $hardcore = (int) request()->query('h', (string) UnlockMode::Softcore);
-$requestedBy = request()->query('z');
+$requestedBy = request()->query('z'); // the z (username) param has been deprecated, but to ensure we don't introduce a breaking change we will keep this here
```

Comment:
`getAchievementDistribution()` takes a `$requestedBy` param, so I left this reference the the z param here in case people continue to call with the z param.

Now that I think about it though, this might be pointless. We could just do `$requested = Auth::user()->User` because as it stands now, the z param will _always*_ be present.

*unless someone is currently calling this endpoint with an empty username, which is possible. The endpoint wouldn't return expected results for them though.

Reply:
I think doing `Auth::user()` is likely preferable to having consumers tack on the `z` query param.